packages/grid/src/vaadin-grid-scroll-mixin.js

```diff
@@ -528,4 +522,16 @@ export const ScrollMixin = (superClass) =>
         this.$.table.style.setProperty('--_grid-horizontal-scroll-position', `${-x}px`);
       }
     }
+
+    /** @private */
+    _areSizerCellsAssigned() {
```

Comment:
What's the reason for having separate implementations for lazy- and non-lazy modes? Would `return this._getColumnsInOrder().every((column) => column._sizerCell);` be sufficient for both modes?

Reply:
Just kept the original since checking one column is already enough for non-lazy. But of course, checking all works for both. Updated the PR to use the same approach regardless of the mode.